### ISSUE TITLE
Install using angular CLI failing

### ISSUE THREAD

**Issue by rtb21** [vmware/clarity#2648]
Having created a new angular project using `ng new clarity-test`, upon running `ng add @clr/angular` as in [the instructions](https://vmware.github.io/clarity/documentation/v0.13/get-started) npm generates the message 

> Cannot read property 'build' of undefined

Using angular v6.1.0, clarity v0.13, node v10.5.0 and npm v6.1.0. for the moment I've therefore dropped back to manually adding clarity.

**Comment by gnomeontherun** [comment]
I just tested without any issues, can you give me the exact commands you used and the resulting output? It helps to use the bug report issue to capture things in an easy format.

**Comment by webmaxru** [comment]
Same error for me

Command:
```
ng add @clr/angular
Installing packages for tooling via npm.
Cannot read property 'build' of undefined
```

Version:
```
ng -v

Angular CLI: 6.2.1
Node: 8.11.3
OS: darwin x64
Angular: 6.1.7
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.8.1
@angular-devkit/build-angular     0.8.1
@angular-devkit/build-optimizer   0.8.1
@angular-devkit/build-webpack     0.8.1
@angular-devkit/core              0.8.1
@angular-devkit/schematics        0.8.1
@angular/cli                      6.2.1
@ngtools/webpack                  6.2.1
@schematics/angular               0.8.1
@schematics/update                0.8.1
rxjs                              6.2.2
typescript                        2.9.2
webpack                           4.18.0
```

**Comment by gnomeontherun** [comment]
Ok confirmed when I use the CLI 6.2.0+, it appears Angular CLI 6.2.x has a breaking change in the angular.json file format and I have been testing on CLI 6.1.x. 

The workaround is to follow the installation guide on the website for now, and I'll get a fix asap.

**Comment by gnomeontherun** [comment]
This looks related, but says its for 7.0 https://github.com/angular/angular-cli/commit/fd7bcd25fe1af58bf25c22968852aa2cbdb4dde1

**Comment by rtb21** [comment]
Probably a bit late now but here's my report for `ng -v` too

```
Angular CLI: 6.2.1
Node: 10.5.0
OS: win32 x64
Angular: 6.1.7
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.8.1
@angular-devkit/build-angular     0.8.1
@angular-devkit/build-optimizer   0.8.1
@angular-devkit/build-webpack     0.8.1
@angular-devkit/core              0.8.1
@angular-devkit/schematics        0.8.1
@angular/cli                      6.2.1
@ngtools/webpack                  6.2.1
@schematics/angular               0.8.1
@schematics/update                0.8.1
rxjs                              6.3.2
typescript                        2.9.2
webpack                           4.18.0
```

**Comment by gnomeontherun** [comment]
Thanks @rtb21 and @webmaxru for the report, I'll have to ensure we test on both 6.1.x and 6.2.x + in the future. This will be fixed after today's release of 0.13.1
